quipucords/tests/api/management/commands/test_create_or_update_user.py

```diff
@@ -45,7 +45,12 @@ def test_create_or_update_user_create_with_valid_password(faker):
     """Test creation of a user with a valid password."""
     out = StringIO()
     username = faker.user_name()
-    password = faker.password(length=settings.QUIPUCORDS_MINIMUM_PASSWORD_LENGTH)
+    password = faker.password(
+        length=settings.QUIPUCORDS_MINIMUM_PASSWORD_LENGTH,
+        digits=True,
+        upper_case=True,
+        lower_case=True,
+    )
```

Comment:
Why change this? It looks like the defaults are already `True`.

https://faker.readthedocs.io/en/master/providers/faker.providers.misc.html#faker.providers.misc.Provider.password
```
password(length: int = 10, special_chars: bool = True, digits: bool = True, upper_case: bool = True, lower_case: bool = True) → str
```

Reply:
funny. I got an local error I assumed was due to these values being false.

Reply:
It's a shame I didn't record the error. It was something along the lines of "password has no numeric characters". 

I'm gonna remove this commit and, shall this error happen again, I'll record it.